Classes/Events/PlaylistEvent.swift

```diff
@@ -0,0 +1,72 @@
+// ===================================================================================================
+// Copyright (C) 2021 Kaltura Inc.
+//
+// Licensed under the AGPLv3 license, unless a different license for a
+// particular library is specified in the applicable library path.
+//
+// You may obtain a copy of the License at
+// https://www.gnu.org/licenses/agpl-3.0.html
+// ===================================================================================================
+//
+//  PlaylistEvent.swift
+//  PlayKit
+//
+//  Created by Sergii Chausov on 27.10.2021.
+//
+
+import Foundation
+
+@objc public class PlaylistEvent: PKEvent {
+    
+    @objc public static let allEventTypes: [PlaylistEvent.Type] = [
+        playListLoaded, playListStarted, playListEnded, playlistCountDownStart, playlistCountDownEnd, playlistLoopStateChanged, playlistLoopStateChanged, playlistAutoContinueStateChanged, playListError, playListLoadMediaError, playListCurrentPlayingItemChanged
```

Comment:
Let's keep all in the same format: 'playlist'.
```suggestion
        playlistLoaded, playlistStarted, playlistEnded, playlistCountDownStart, playlistCountDownEnd, playlistLoopStateChanged, playlistLoopStateChanged, playlistAutoContinueStateChanged, playlistError, playlistLoadMediaError, playlistCurrentPlayingItemChanged
```